examples/bunny-twirl/index.js

```diff
@@ -56,13 +56,26 @@ container.addChild(text)
 let isAnimating = true
 let isPlaying = false
 
+// Generate map
+const challengeNumber = Rune.getChallengeNumber();
+// const nextChallengeNumber = challengeNumber + 1;
+
+// 1. Game map randomization
+const random = new Math.seedrandom(challengeNumber)
+const pseudoRandomNumber = random()
+const speed = (pseudoRandomNumber / 100) * 3 + 0.005
```

Comment:
Is the `+ 0.005` to ensure speed is at least 0.005? If so, using `Math.min` instead would make that clear.

Reply:
Let me use the standard way as per https://stackoverflow.com/questions/4959975/generate-random-number-between-two-numbers-in-javascript to make it more clear.

Reply:
Fixed in https://github.com/rune/rune-games-sdk/pull/24/commits/d3f340842c4b95f6588463bf94299bc7357406b7